packages/components/src/FormatFile/FormatFile.css

```diff
@@ -51,12 +51,61 @@
   margin: 0 auto;
 }
 
-.progress {
+.fileContentWrapper {
```

Comment:
This is new to the component, it looks like in the original `FormatFile--imageBlock` handled both the image or the file styling. This allowed for simpler styling and markup as there didn't need to be a fileContentWrapper inside the image block. In the update this fileContentWarpper only applies to the thumbnail variant and not the original.

![Screen Shot 2022-05-25 at 10 35 52 AM](https://user-images.githubusercontent.com/10064579/170315326-8623838e-6e32-4b92-98f3-5486c5aecf8e.png)